lib/operationTimeoutError.ts

```diff
@@ -12,7 +12,7 @@ export class OperationTimeoutError extends Error {
   /**
    * Provides a logical amqp error condition.
    */
-  readonly condition: string = "amqp:operation-timeout";
+  readonly condition: string = "rhea-promise:operation-timeout";
```

Comment:
I would suggest that this error should not have a `condition` property to begin with.

The way this error is used rhea-promise is to indicate operation timeout. Operation Timeout is a generic error and not AMQP error. With the `condition` property, `isAmqpError()` would return `true` for this error where it shouldn't, because this error has nothing to do with amqp.

The fact that `isAmqpError()` returning `true` helps us in `amqp-common` library should not be a driving factor here.